xbuild/src/lib.rs

```diff
@@ -296,7 +296,7 @@ impl CompileTarget {
 
     pub fn rust_triple(self) -> Result<&'static str> {
         Ok(match (self.arch, self.platform) {
-            (Arch::Arm64, Platform::Android) => "aarch64-linux-android",
+            (_, Platform::Android) => self.ndk_triple(),
```

Comment:
`x86_64` was missing here, so I figured we could just reuse `ndk_triple` since they're the same?

Reply:
well, if we add more targets it might not be. I'd prefer to not reuse `ndk_triple` here

Reply:
I'll add the missing `x86_64-linux-android` arch separately then.